app/assets/stylesheets/framework/form.scss

```diff
@@ -0,0 +1,44 @@
+// cleaning bootstrap. Can be removed once bootstrap will be stripped out
+label {
+  margin-bottom: 0;
+}
+// end cleaning bootstrap
+
+button,
+input,
+select,
+textarea {
+  // Fix dirty box model rendering on inputs
+  -moz-box-sizing: border-box;
```

Comment:
- Properties should be ordered background-color, border, border-radius, -moz-box-sizing, -webkit-box-sizing, box-sizing, font-size, padding
- Avoid vendor prefixes.